pkg/controller/volume/attach_detach_controller.go

```diff
@@ -205,7 +207,7 @@ func (adc *attachDetachController) nodeAdd(obj interface{}) {
 	}
 
 	nodeName := node.Name
-	if _, exists := node.Annotations[attachdetach.ControllerManagedAnnotation]; exists {
+	if _, exists := node.Annotations[volumehelper.ControllerManagedAnnotation]; exists {
```

Comment:
I think the name of this const is now ambiguous since it moved out of attach-detach package -- call it `ControllerManagedAttachAnnotation` ?